### ISSUE TITLE
Syncer does not throw on empty ipfs meta

### ISSUE THREAD

**Issue by fbielejec** [district0x/memefactory#362]
There is meme with empty meta-hash: http://qa.district0x.io/meme-detail/0xb434ce89a91ad414d585cf8b7bc28866033013dd

```
{"regEntry_address":"0xb434ce89a91ad414d585cf8b7bc28866033013dd",
"meme_metaHash":"QmYRz73zPLsdWpbWG3yPMvfbeQNipyTyDM5AosTnSP1Sau",
"meme_imageHash":""}
```

This content indeed doesn't not exist on the network as both requests time out:

> curl "http://qa.district0x.io:5001/api/v0/object/get?arg=QmYRz73zPLsdWpbWG3yPMvfbeQNipyTyDM5AosTnSP1Sau&encoding=json"

> curl "http://ipfs.io/api/v0/object/get?arg=QmYRz73zPLsdWpbWG3yPMvfbeQNipyTyDM5AosTnSP1Sau&encoding=json"

Yet when re-syncing no errors are ever thrown/logged.

**Comment by fbielejec** [comment]
Good place to enforce healthy ipfs-meta is the already existing sanity check: https://github.com/district0x/memefactory/blob/master/src/memefactory/server/syncer.cljs#L151

**Comment by fbielejec** [comment]
This was fixed, accidentally with #486. 

https://ipfs.qa.district0x.io/gateway/ipfs/QmYRz73zPLsdWpbWG3yPMvfbeQNipyTyDM5AosTnSP1Sau

Here's an error thrown:
https://sentry.io/organizations/district0x/issues/1000764949/?project=1306960&referrer=slack
